src/main/java/sg/ncl/MainController.java

```diff
@@ -145,36 +146,40 @@ public String calendar1() {
     }
 
     @RequestMapping(value="/futureplan/download", method=RequestMethod.GET)
-    public void futureplanDownload(HttpServletResponse response) throws FuturePlanDownloadException {
+    public void futureplanDownload(HttpServletResponse response) throws FuturePlanDownloadException, IOException {
+        InputStream stream = null;
         response.setContentType("application/pdf");
         try {
             File fileToDownload = new File("src/main/resources/downloads/future_plan.pdf");
-            InputStream is = new FileInputStream(fileToDownload);
+            stream = new FileInputStream(fileToDownload);
             response.setContentType("application/force-download");
             response.setHeader("Content-Disposition", "attachment; filename=future_plan.pdf");
-            IOUtils.copy(is, response.getOutputStream());
+            IOUtils.copy(stream, response.getOutputStream());
             response.flushBuffer();
-            is.close();
         } catch (Exception ex) {
             logger.info("Error writing file to output stream.");
             throw new FuturePlanDownloadException("IOError writing file to output stream");
+        } finally {
+            stream.close();
         }
     }
 
     @RequestMapping(value="/OrderForm_v1/download", method=RequestMethod.GET)
-    public void OrderForm_v1Download(HttpServletResponse response) throws OrderFormDownloadException {
+    public void OrderForm_v1Download(HttpServletResponse response) throws OrderFormDownloadException, IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: sg.ncl.exceptions.OrderFormDownloadException, java.io.IOException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1160)